ruby/computer_science/project_linked_lists.md

```diff
@@ -56,11 +56,34 @@ Build the following methods in your linked list class:
 1. `#to_s` represent your LinkedList objects as strings, so you can print them out and preview them in the console.
     The format should be: `( value ) -> ( value ) -> ( value ) -> nil`
 
-### Extra credit
+#### Extra credit
 
 1. `#insert_at(value, index)` that inserts a new node with the provided `value` at the given `index`.
 1. `#remove_at(index)` that removes the node at the given `index`.
 
 **Extra Credit Tip:** When you insert or remove a node, consider how it will affect the existing nodes. Some of the nodes will need their `#next_node` link updated.
 
+#### Test it out
+
+Let's test out the Linked List you made!
+
+1. Create a `main.rb` file and make sure it inherits the file containing `LinkedList`. This is where we'll test the list.
```

Comment:
```suggestion
1. Create a `main.rb` file and make sure it requires the file containing `LinkedList`. This is where we'll test the list.
```
Recommend changing this from "inherits" (which generally refers to subclassing) to "requires" (which is the Ruby terminology for loading different files)